services/ux-backend/main.go

```diff
@@ -94,3 +106,22 @@ func main() {
 	log.Fatal(err)
 
 }
+
+func newClient() (client.Client, error) {
+	klog.Info("Setting up k8s client")
+	scheme := runtime.NewScheme()
+	if err := corev1.AddToScheme(scheme); err != nil {
+		return nil, err
+	}
+
+	config, err := config.GetConfig()
+	if err != nil {
+		return nil, err
+	}
+	k8sClient, err := client.New(config, client.Options{Scheme: scheme})
+	if err != nil {
+		return nil, err
+	}
+
+	return k8sClient, nil
+}
```

Comment:
I believe we already have other places that create a new client. 
Please reuse the same code instead of rolling out your own

Reply:
The newClient() function is used in onboarding-validation-keys-generator. I have re-used the same code, not modified anything.